packages/wallet-sdk/src/sign/scw/SCWSigner.ts

```diff
@@ -12,7 +8,7 @@ import { AppMetadata, ProviderEventCallback, RequestArguments } from ':core/prov
 import { WalletConnectResponse } from ':core/rpc/wallet_connect.js';
 import { Address } from ':core/type/index.js';
 import { ensureIntNumber, hexStringFromNumber } from ':core/type/util.js';
-import { createClients, SDKChain } from ':store/chain-clients/utils.js';
+import { SDKChain, createClients } from ':store/chain-clients/utils.js';
```

Comment:
Interesting that this import updated. I believe we have import sorting enabled for biome

Reply:
Maybe this?
```
>>> ord("S")
83
>>> ord("c")
99
```

Reply:
Most likely. all good